src/api/app/controllers/status_project_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StatusProjectController < ApplicationController
+  def show
+    dbproj = Project.get_by_name(params[:project])
+    @packages = ProjectStatus::Calculator.new(dbproj).calc_status
+    find_relationships_for_packages(@packages)
```

Comment:
This is still in [status_controller.rb](https://github.com/openSUSE/open-build-service/blob/873a6afe9e609178387c450c606d8cf616ccf664/src/api/app/controllers/status_controller.rb#L50)

Reply:
missing coffee.. everything must be moved to the status_project_controller :P

Reply:
BOOOOOM! fixed